src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -379,6 +379,23 @@
                         }
                       }
                       """),
+              @ExampleObject(
+                  name = "createCollectionLexical",
+                  summary = "`CreateCollection` command with lexical indexing ($lexical) enabled",
+                  value =
+                      """
+                                {
+                                  "createCollection": {
+                                      "name": "events",
+                                      "options": {
+                                          "lexical": {
+                                              "enabled": true,
```

Comment:
I'm curious, why is `enabled: true` necessary, vs assuming that presence of `lexical` means `enabled: true` vs absence of `lexical` means lexical is not enabled? Aaron mentioned this on our call yesterday but I didn't get any details as to why.

Reply:
I think it's due to multiple factors. First: the default without "lexical" should be to enable, so to disable, one needs some value. Use of empty Object (`{ }`) seems bit confusing (difference b/w empty and missing being sort of semantically small).
Hence to disable, minimally:

```
{ "lexical": { "enabled": false  }}
```

and then it would be least confusing to always require specifying `enabled` if any value added.

Alternative would be to infer existence from combination of things.

So I think this is mostly due to defaulting to Lexical being enabled unless explicitly declined.

Reply:
Ok I think I understand, the issue is that it is perfectly valid to enable `lexical` with no options, but `lexical: {}` would break the pattern we have of "empty options same as omitting options". If that's the case, makes sense :+1:

Reply:
Right, it's a tough call. I think we are breaking the pattern, due to starting to use `enabled` flag as a basic mandatory thing.